ROADMAP.md

```diff
@@ -4,25 +4,25 @@ This roadmap outlines the planned development of the AI Experiments plugin from
 
 ## Milestone 0.1.0 - Plugin Foundations and First Feature Integration
 
-**Target:** Early October 2025
+**Target:** Late November 2025 (Aligned with WordPress 6.9 Release)
 
 ### Goals
 
-- [ ] Scaffold the initial plugin structure in the `WordPress/ai` GitHub repository ([#24](https://github.com/WordPress/ai/issues/24))
-- [ ] Create a basic admin settings screen with a toggle for enabling experimental features ([#25](https://github.com/WordPress/ai/issues/25))
+- [X] Scaffold the initial plugin structure in the `WordPress/ai` GitHub repository ([#24](https://github.com/WordPress/ai/issues/24))
+- [X] Create a basic admin settings screen with a toggle for enabling experimental features ([#25](https://github.com/WordPress/ai/issues/25))
 - [ ] Integrate the WP AI Client SDK (and underlying PHP AI Client SDK) with a simple service provider setup ([#26](https://github.com/WordPress/ai/issues/26))
```

Comment:
I think you can technically mark this as done, as the Title Generation Experiment does integrate with the WP AI Client

Reply:
If I check this branch out, I'm not seeing `Integrate the WP AI Client SDK (and underlying PHP AI Client SDK) with a simple service provider setup` as checked. Not a big deal though, just seems like technically that piece is done

Reply:
Updated in [83e941b](https://github.com/WordPress/ai/pull/106/commits/83e941b28eb44d3e78bb1b9f27df37cfe5baa841).